app/api/domains/api.py

```diff
@@ -760,7 +760,8 @@ async def api_get_match(
     """Return information of a given multiplayer match."""
     # TODO: eventually, this should contain recent score info.
 
-    if not (match := app.state.sessions.matches[match_id]):
+    match = app.state.sessions.matches[match_id]
```

Comment:
lmao this syntax is so unintuitive, what was i thinking

Reply:
oh right, i should change that to get()

Reply:
to be fair i think `api_get_match` should be refactored because it looks like a dumpster fire rn

Reply:
yeah i'm actually going to revert the commit you added - there are multiple additional uses of this and we can leave this for another pr

Reply:
okok